BrainPortal/app/views/data_providers/_one_data_provider_table.html.erb

```diff
@@ -76,15 +76,19 @@
   %>
 
   <% t.column("Alive?", :alive) do |dp| %>
-    <% if dp.online? %>
-      <%= staggered_loading "span", url_for(:action  => :is_alive, :id  => dp.id), :error_message  => "<span class=\"warning\">???</span>" do %>
-        <span class="loading_message"><%= html_colorize("(...)","magenta") %></span>
+    <%= on_click_ajax_replace( { :element  => "div",
```

Comment:
If the DP is not online (an attribtue of the model) we don't nee dto do a check, we can already fill the row's information, That's why there was a IF int he original code.